smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructuredMemberWriter.java

```diff
@@ -65,6 +70,131 @@ void writeMembers(TypeScriptWriter writer, Shape shape) {
         }
     }
 
+    /**
+     * Recursively writes filterSensitiveLog for arrays (CollectionShape)
+     */
+    void writeFilterSensitiveLogForCollection(TypeScriptWriter writer, MemberShape collectionMember) {
+        Shape memberShape = model.expectShape(collectionMember.getTarget());
+        if (memberShape instanceof StructureShape) {
+            // Call filterSensitiveLog on Structure
+            writer.write("${T}.filterSensitiveLog", symbolProvider.toSymbol(collectionMember));
```

Comment:
It doesn't look like you're using sections for these, so the variable writes should be without braces (`$T` instead of `${T}`, etc.) This occurs in multiple places in this update.

Reply:
Removed braces for variable writes in commit https://github.com/awslabs/smithy-typescript/pull/170/commits/4dde9173ce986f26f14d1f63c7f9db3158883cff